tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_parent_test.dart

```diff
@@ -25,60 +24,6 @@ void main() {
           .validateAll();
 
       expect(collector.errors, isNotEmpty);
-
```

Comment:
chore: This file still contains a bunch of tests that are not required anymore. Please remove all tests except maybe a single one that validates that the parent keyword generates an error deprecation warning.

You have basically removed the expects from all tests in the file that make them unique, so we are just validating the same thing over and over again.